cocnept-designs/v2/index.html

```diff
@@ -13,15 +13,42 @@
         <div class="screen" id="s-smallscreen">
             <p>Bruh. Your screen is too small.<br>Responsive Design coming sometime</p>
         </div>
+        
         <div id="container">
+            <div class="screen" id="s-menu"></div>
+            <div id="menu">
+                <div class="menurow">
+                    <button>Save</button>
+                    <button>Leaderboard</button>
+                </div>
+                <p class="menutitle">Sounds</p>
+                <div class="menutable">
+                    <div class="mtlabel">
+                        <label for="effects">Effects</label>
+                        <label for="music">Music</label>
+                    </div>
+                    <div class="mtinput">
+                        <input type="range" id="effects">
+                        <input type="range" id="music">
+                    </div>
+                </div>
+                <div class="menurow">
+                    <button onclick="cambiar('unmenu')">Exit Options</button>
+                    <button>Exit Game</button>
+                </div>
+                <div class="menurow">
+                    <button>Report a Bug</button>
+                </div>
+            </div>
+            
             <div id="controls">
                 <p>Mon 05/12/20</p>
                 <p>10:00 AM</p>
                 <p>$500</p>
                 <div class="stretch"></div>
-                <button>Computer</button>
-                <button>Office</button>
-                <button>Menu</button>
+                <button onclick="cambiar('game')">Computer</button>
+                <button onclick="cambiar('office')">Office</button>
+                <button onclick="cambiar('menu')">Menu</button>
```

Comment:
lol why Spanish

Reply:
switch is a keyword lol
didn't see this comment till now lol

Reply:
lol why not `switchMenu` whatever I already changed it to `switchMenu`

Reply:
idk I always prefer one-word names

Reply:
oof